core/test/test_move_to.cpp

```diff
@@ -62,7 +62,7 @@ TEST_F(PandaMoveTo, mapTarget) {
 }
 
 TEST_F(PandaMoveTo, stateTarget) {
-	move_to->setGoal([]() {
+	move_to->setGoal([] {
```

Comment:
Interesting. Haven't seen this syntax before :wink:

Reply:
Me neither, but it's nice. The downside is that for some reason `[] -> Type {}` is not supported syntax though and requires `[]() -> Type {}` (which reads so much... better?)